File: Articles/A comparsion between C, C++ And rust binary/src.c
#include <stdio.h>
#include <string.h>

int main(int argc, char **argv)
{
    char buf[100];
    
    printf("Enter password: ");

    scanf("%99s", &buf);

    if (!strcmp(buf, "p@ssw0rd"))
    {
        printf("Correct!\n");
    }
    else
    {
        printf("Incorrect...\n");
    }

    return 0;
}

File: Articles/A comparsion between C, C++ And rust binary/src.rs
use std::io::{stdin};

fn main()
{
    let mut pass = String::new();

    stdin().read_line(&mut pass);

    if pass.trim() == "p@ssw0rd"
    {
        println!("Correct!");
    }
    else
    {
        println!("Incorrect...");
    }
}
